packages/main/src/MultiInput.ts

```diff
@@ -289,8 +289,10 @@ class MultiInput extends Input {
 		const cursorPosition = this.getDomRef()!.querySelector(`input`)!.selectionStart;
 		const tokens = this.tokens;
 		const lastToken = tokens.length && tokens[tokens.length - 1];
+		const isNumberOrEmail = this.type === "Number" || this.type === "Email";
 
-		if (cursorPosition === 0 && lastToken) {
+		// Selection is only permitted with text/search, URL, tel and password.
```

Comment:
I would change this text to reflect the condition below - e.g. "cursor position can only be determined for .....". Or take the explanation from the MDN: "selectionStart property applies only to inputs of types text, search, URL, tel...". You can also check if (cursorPosition == null && !this.value) with some explanation when it is null. Thus we will not need isNumberOrEmail variable